src/components/Book/Book.js

```diff
@@ -37,16 +38,15 @@ export const Book = (props) => {
     cancelOrder,
     onRowClick,
     className,
-    parentWidth,
```

Comment:
parentWidth was coming from grid comp available space, and was added to resolve issue of book layout continuously changing between mobile/non-mobile layout. But I verified and not able to repro the issue, so we can now remove this 👍